packages/ui/src/working-groups/components/WorkingGroupListItem.tsx

```diff
@@ -44,7 +44,7 @@ export function WorkingGroupListItem({ group }: WorkingGroupProps) {
           <GroupTitle as={GhostRouterLink} to={groupAddress}>
             {nameMapping(group.name)}
           </GroupTitle>
-          <GroupTooltip link={workingGroupLinks[group.id]} show={showTooltip} />
+          <GroupTooltip link={workingGroupLinks[group.id]} show={true} />
         </Flex>
```

Comment:
Looks like this can be simplified again to replace the `Flex`:
```
          <GroupTitle as={GhostRouterLink} to={groupAddress}>
            {nameMapping(group.name)}
            <GroupTooltip link={workingGroupLinks[group.id]} />
          </GroupTitle>
```

Reply:
sorry for misleading you here, realized we can use the `Tooltip` component and do away with `GroupTooltip.tsx` altogether. Just tested locally:
```
      <GroupContentBlock>
        <Tooltip
          tooltipTitle={wgNameMapped}
          tooltipText=""
          tooltipLinkURL={wgLink}
          tooltipLinkText="Learn more about this group"
        >
          <GroupTitle as={GhostRouterLink} to={groupAddress}>
            {wgNameMapped}
          </GroupTitle>
        </Tooltip>

        <GroupContent as={GhostRouterLink} to={groupAddress}>
          {subtitleMapping(group.name)}
        </GroupContent>
      </GroupContentBlock>
```
![group-tooltip](https://user-images.githubusercontent.com/31551045/197806366-fe6234b4-39c5-4093-b1dc-fb8bbccfadfe.png)